src/Preview.tsx

```diff
@@ -229,6 +229,14 @@ const Preview: React.FC<PreviewProps> = props => {
     ],
   );
 
+  const onDoubleClick = () => {
+    if (!visible) {
+      return;
```

Comment:
```suggestion
    if (visible) {
      ...
    }
```

Can we use the initial approach to keep the test coverage 100%?

Reply:
That is OK.

Reply:
Got it, pushed a commit!